pkg/mcptools/deploytools.go

```diff
@@ -69,8 +123,35 @@ func (d *DeployTools) Init(mcpServer *server.MCPServer) {
 		Tool: mcp.NewTool(
 			DeployToolName,
 			mcp.WithDescription(`
-Deploys TSSC components to the cluster, uses the cluster configuration to deploy
-the TSSC components sequentially.`,
+Deploys TSSC components to the cluster, using the cluster configuration to deploy
+the components sequentially. Note the "dry-run" flag: the deployment process will
+only be initiated when the "dry-run" flag is set to "false". By default, this flag
```

Comment:
I find it strange that dry-run is the default. I can't think of another tools that is setup this way.